deploy/charts/templates/deployment.yaml

```diff
@@ -29,8 +29,8 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}" # In Jenkins, need to call helm with --set image.tag=master-XY
-        command: ["/usr/bin/npm"]
-        args: ["start", "--", "--port=8888", "--config={{ .Values.deployenv.environment }}", "--prefix", "/ui/ui-build"]
+        command: ["npm"]
+        args: ["start", "--", "--port=8888", "--config={{ .Values.deployenv.environment }}", "--prefix", "/ui"]
```

Comment:
Likewise i recommend a little streamlining here. We should decided whether we want this to live in the deployment spec or Dockerfile -- but I'd strongly recommend against having this in both places.

The Dockerfile is OK if we don't need anything dynamic or can rely on environment variables.

If we have a more complex dynamic scenario then doing this via Helm + values in the deployment spec might be easier.

Reply:
Agree, I'm totally fine with moving the command from here if it gets handled by the other dynamic config...